src/datasources/jwt/configuration/jwt.configuration.module.ts

```diff
@@ -5,6 +5,9 @@ import { ConfigModule } from '@nestjs/config';
 import { NestConfigurationService } from '@/config/nest.configuration.service';
 import jwtConfiguration from '@/datasources/jwt/configuration/jwt.configuration';
 
+/**
+ * @deprecated
+ */
```

Comment:
We don't need to deprecate this as it can be removed.

Reply:
The idea was to keep it so that we can decide whether we want to remove all the localized configuration values or not. But as you mentioned, let's delete it for now as we can always revert the changes.
Done
https://github.com/safe-global/safe-client-gateway/pull/2067/commits/196db4fd3bc702923e7102e99ad69fef810087f7